ci/docker_report.md.tpl

```diff
@@ -0,0 +1,42 @@
+## Docker Preview Image :whale:
+
+A preview docker image for this branch is available with the following tag:
+
+```
+$PREVIEW_TAG
+```
+
+## Use at your own risk :warning:
+
+Preview builds make no promises about stability or feature completeness.  Use them at your own risk. A preview build is not generated if tests failed,  so they have at least passed the common test suite. They may or may not have  been subjected to the asynchronous stress test and chaos pipelines.
+
+## Usage :newspaper:
+
+### Docker-compose
+
+You can obtain a [docker-compose.yaml here](https://weaviate.io/developers/weaviate/current/installation/docker-compose.html#configurator) and configure it to your liking. Then make sure to set `services.weaviate.image` to `$PREVIEW_TAG'`. For example, like so:
```

Comment:
\`$PREVIEW_TAG'\`
unnecessary `'`

Reply:
ah, sorry, I missed this. I'll fix it on the merged branches